deep-learning/_papers/2017-02-28-you-only-look-once.md

```diff
@@ -0,0 +1,52 @@
+
+
+```
+---
+layout: review
+title:  You Only Look Once: Unified, Real-Time Object Detection (YOLO)
+author: Frédéric Branchaud-Charron
+link:   https://arxiv.org/abs/1506.02640
+tags:   convolutional-networks deep-learning object-detection bounding-boxes
+---
+```
+
+YOLO v1
+
+## Summary
+
+**YOLO** is an unified solution to the object detection problem. Other approches like **R-CNN** are using a two step process *(localization and classification)* where **YOLO** is a one step process. Also, **Fast YOLO** (or tiny-yolo) is achieving of 155 fps where **Faster R-CNN** is doing 18 fps.
+
+On the Pascal VOC-2012 dataset, **YOLO** is achieving 63.4 mAP where **Faster R-CNN** is doing 73.2. In this case, **Faster R-CNN** is not real-time (7 fps) while **YOLO** is doing around 60 fps.
```

Comment:
A table would be nice for this kind of comparison. Something like this:
```
|--------------+--------------------+-----|
| Model        | Mean avg precision | FPS |
|:-------------|-------------------:|----:|
| Faster R-CNN | 73.2               | 7   |
| YOLO         | 63.4               | 60  |
|--------------+--------------------+-----|
```
Notes

* Notice the colons `:`. They indicate alignment.
* There is no css style for tables yet. I will fix this soon.